src/utils/sort.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright © 2020 The PES Open Source Team pesos@pes.edu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"sort"
+	"strconv"
+)
+
+// SortData helps sort table rows. It sorts the table based on values given in the sortIdx column and sorts ascending if sortAsc is true. sortCase is set to identify the set of 'less' functions to use to sort the selected column by.
```

Comment:
Small nit, but maybe break this doc comment into multiple lines?

Reply:
But why? It's clean when reading the doc this way right?

Reply:
Not really, you have to scroll and all, gets weird, maybe it's neat cause it wraps it in your editor?

Reply:
So newlines would fix the scroll? This is weird :P

Reply:
Don't add new lines, break the doc comment into multiple lines to decrease length per line XD

Reply:
Yea I got it xD